dev-requirements.txt

```diff
@@ -12,7 +12,6 @@ coverage==5.3             # via -r dev-requirements.in
 flake8-black==0.2.1       # via -r dev-requirements.in
 flake8-isort==4.0.0       # via -r dev-requirements.in
 flake8==3.8.3             # via -r dev-requirements.in, flake8-black, flake8-isort
-importlib-metadata==1.7.0  # via -c requirements.txt, flake8, pluggy, pytest
```

Comment:
I ran `pip-compile dev-requirements.in`.

Reply:
Maybe because I ran it under Python 3.8? I can revert the change if seems risky.